### ISSUE TITLE
Addon doesn't work with firefox and testem

### ISSUE THREAD

**Issue by dennismende** [kategengler/ember-cli-code-coverage#98]
Testem file:
`module.exports = {
  framework: 'qunit',
  test_page: 'tests/index.html?hidepassed',
  disable_watching: true,
  launch_in_ci: [
    'PhantomJS',
    'Firefox',
  ],
  launch_in_dev: [
    'PhantomJS',
    'Firefox',
  ],
};`

When I start ember testmode with `COVERAGE=true ember test` then all tests get executed with phantom but then it hangs up when firefox should get started.

Without `COVERAGE=true` everything works fine.

Ember / Data / CLI = 2.11.0

**Comment by dennismende** [comment]
I will setup a clean install and try it out. Will share the results later today.

**Comment by rwjblue** [comment]
Awesome, thank you!

**Comment by dennismende** [comment]
Tried it out with a new repo and everything works fine. Could be an issue together with some other addons. I will try to find out which addon causes the issue here.

**Comment by Turbo87** [comment]
We've hit a similar issue in one of our client apps at @simplabs and created a slimmed down test case at https://github.com/Turbo87/__test-stall-repro

tl;dr the missing `server.shutdown()` call appears to keep the test process alive when running with `COVERAGE=true ember test`. the strange thing is that it seems to work as expected when just running `ember test` without coverage.

**Comment by Turbo87** [comment]
cc @samselikoff maybe you have a clue on what's going on and how to resolve this

**Comment by samselikoff** [comment]
hm - is it mirage-related? `server.shutdown()` from mirage right? I don't see Mirage in use here

**Comment by Turbo87** [comment]
- https://github.com/Turbo87/__test-stall-repro/blob/master/tests/integration/services/foo-test.js#L4-L15
- https://github.com/Turbo87/__test-stall-repro/blob/master/mirage/config.js#L27

those two are the important parts

**Comment by Turbo87** [comment]
https://github.com/ember-cli/ember-cli/issues/6773 could be related too

**Comment by rwwagner90** [comment]
@Turbo87 is this still an issue? Lots of things worked themselves out in the refactor. Can you please try 1.0.0-beta.2?

**Comment by rwwagner90** [comment]
@Turbo87 @dennismende is this still an issue as of the latest beta, 1.0.0-beta.3?

**Comment by Turbo87** [comment]
closing this for now. will reopen if I should ever hit that issue again in the future.
